packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -346,6 +346,18 @@ type PermissionsUpdated = BaseTrack & {
     };
 };
 
+type FieldValueSearch = BaseTrack & {
+    event: 'field_value.search';
+    userId?: string;
+    properties: {
+        projectId: string;
+        fieldId: string;
+        searchCharCount: number;
+        resultsCount: number;
+        searchLimit: number;
```

Comment:
Is `seachLimit` the query limit that we've set for how many results show?

Reply:
correct 👍